consensus/istanbul/core/core.go

```diff
@@ -342,7 +356,7 @@ func (c *core) startNewRound(round *big.Int) {
 	}
 	c.newRoundChangeTimer()
 
-	logger.Debug("New round", "new_round", newView.Round, "new_seq", newView.Sequence, "new_proposer", c.valSet.GetProposer(), "valSet", c.valSet.List(), "size", c.valSet.Size(), "isProposer", c.isProposer())
+	logger.Debug("New round", "new_round", newView.Round, "new_seq", newView.Sequence, "new_proposer", c.current.Proposer(), "valSet", c.current.ValidatorSet().List(), "size", c.current.ValidatorSet().Size(), "isProposer", c.isProposer())
```

Comment:
Can `c.current` be nil at this point?  If so, there would be a seg fault as it's methods are used multiple points in the log message.

Reply:
No because on line 348 on this same method we call `resetRoundState` that set `c.current`